MMM-Fuel.js

```diff
@@ -277,10 +308,31 @@ Module.register("MMM-Fuel", {
         appendTo.appendChild(row);
 
         if(this.config.showAddress){
+			function titleCase(str) {
```

Comment:
this function should be a module function outside of getDom

titleCase: function(str){
...
}

usage in getDom -> this.titleCase(...);